.circleci/scripts/binary_populate_env.sh

```diff
@@ -100,6 +100,11 @@ if [[ "$PACKAGE_TYPE" == libtorch ]]; then
   POSSIBLE_JAVA_HOMES+=(/usr/local)
   POSSIBLE_JAVA_HOMES+=(/usr/lib/jvm/java-8-openjdk-amd64)
   POSSIBLE_JAVA_HOMES+=(/Library/Java/JavaVirtualMachines/*.jdk/Contents/Home)
+  # Using 8.3 filenames is a bit annoying...
+  POSSIBLE_JAVA_HOMES+=(/c/Progra~1/Java/jdk*)
```

Comment:
Why do we need to use 8.3 filenames here? Does the following one work?
```
POSSIBLE_JAVA_HOMES+=("/c/Program Files/Java/jdk*")
POSSIBLE_JAVA_HOMES+=("/c/Program Files/OpenJDK/jdk*")
POSSIBLE_JAVA_HOMES+=("/c/Program Files (x86)/Java/jdk*")
POSSIBLE_JAVA_HOMES+=("/c/Program Files (x86)/OpenJDK/jdk*")
```

Reply:
I had tried those, but it wasn't working locally when debugging. It seems like it's not getting picked up in any case.